awx/ui_next/src/screens/Inventory/InventoryHostDetail/InventoryHostDetail.test.jsx

```diff
@@ -32,6 +32,9 @@ describe('<InventoryHostDetail />', () => {
       assertDetail('Description', 'localhost description');
       assertDetail('Created', '10/28/2019, 9:26:54 PM');
       assertDetail('Last Modified', '10/29/2019, 8:18:41 PM');
+      expect(wrapper.find(`Detail[label="Activity"] Sparkline`)).toHaveLength(
```

Comment:
Since you are manipulating the behavior of `<Detail/>` it makes sense to add another test that asserts that this item is not there if there are no recent jobs.

Reply:
👍 Added test coverage to cover this here: eec217731558b9480e0c436103106b1a7510d146